styles/QuizSelection.module.scss

```diff
@@ -0,0 +1,74 @@
+.container {
+  min-height: 100vh;
+  padding: 0 0.5rem;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-evenly;
+  align-items: center;
+  min-height: 100vh;
+  background: radial-gradient(#3a445c, var(--dark-shades));
+}
+
+@media screen and (max-width: 321px) {
+  .container {
+    padding: 0;
+  }
+}
+.main {
```

Comment:
Consider renaming this main-content or something else along those lines,

Having a class named main and an HTML element named main here can sometimes trip us up.

you can also just select the main HTML item here if that's what you're going for.

If that doesn't work, you can add this main class CSS into the <main> containing object using the main -element selector.

Reply:
Ok I renamed it to mainContent @angelo do you want me to also change how main is named in the home module and index file?

Reply:
If they're named that then yea that would be awesome 😎